Emby.Server.Implementations/HttpServer/WebSocketConnection.cs

```diff
@@ -209,6 +209,12 @@ private async Task ProcessInternal(PipeReader reader)
                 return;
             }
 
+            if (stub is null)
+            {
+                _logger.LogError("Error processing web socket message");
```

Comment:
This should never happen AFAIK?

Reply:
in .net5 the jsonserializer always returns nullable objects